NASA-AMMOS/aerie-ui
# Add permissions to Activity Directives Table and Types Panels

Resolves #751 

Addendum: Resolves #759 

To test:
1. Create a plan with directives
2. Log out and log in as a different user and swap the role to "user"
3. Navigate to the plan that was just created
4. Verify that directives cannot be added via the "+" icon or dragging them from the Activity Types panel on to the timeline
5. Verify that directives cannot be deleted via the Directives Table panel
